src.mk

```diff
@@ -300,6 +286,26 @@ LIB_SOURCES_ASM =
 LIB_SOURCES_C =
 endif
 
+# range_tree is not compatible with non GNU libc on ppc64
+# see https://jira.percona.com/browse/PS-7559
+ifneq ($(PPC_LIBC_IS_GNU),0)
+LIB_SOURCES +=\
```

Comment:
The src.mk file is included before the make_platform.mk file.  Is PPC_LIBC_IS_GNU going to be set properly before the platform file is generated?

Reply:
Indeed. I have fixed that now.